Test/MySafePriorityQueue.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Test;
+
+public class MySafePriorityQueue<T, K> where K : IComparable<K>
```

Comment:
```suggestion
public class MySafePriorityQueue<Value, Priority> where Priority : IComparable<Priority>
```
Имена типовых переменных - это имена. Поэтому они должны быть содержательными.

Reply:
На кр по котлину в прошлом году не было такого требования на имена типов в аналогичном задании (только без многопоточки), где была очередь с приоритетом. https://learn.microsoft.com/ru-ru/dotnet/csharp/fundamentals/coding-style/coding-conventions тут не нашла такого требования. В каком источнике можно найти +- полный список требований к кодстайлу?

Reply:
-0.5

Reply:
Так вы же сказали, что не будете за это снижать..